lightify/__init__.py

```diff
@@ -917,8 +917,12 @@ def __init__(self, host, new_device_types=None):
         self.__device_types.update(new_device_types or {})
 
         self.__logger = logging.getLogger(MODULE)
-        self.__logger.addHandler(logging.NullHandler())
-        self.__logger.info('Logging %s', MODULE)
+        self.__logger.setLevel(logging.INFO)
+        if loghandler is not None:
```

Comment:
Just `if loghandler:`?
```suggestion
        if loghandler:
```